c2corg_ui/static/js/documentediting.js

```diff
@@ -679,5 +681,50 @@ app.DocumentEditingController.prototype.setOrientation = function(orientation, d
 }
 
 
-app.module.controller('appDocumentEditingController',
-    app.DocumentEditingController);
+/**
+ * @param {Object} doc
```

Comment:
What's the connection between the editing and the simple search?
I don't think simple search results are the same than regular "get-document" responses.